specifications/api/swagger.yaml

```diff
@@ -914,6 +914,69 @@ components:
         id: *addressId
         state: *addressState
 
+    ApiAddressInspect: &ApiAddressInspect
+        type: object
+        required:
+        - address_style
+        - stake_reference
+        properties:
+          address_style:
+            type: string
+            enum:
+            - Shelley
+            - Icarus
+            - Byron
+          stake_reference:
+            type: string
+            enum:
+            - none
+            - by value
+            - by pointer
+          network_tag:
+            description: Can be null for 'Icarus' and 'Byron' styles.
+            type: integer
+            minimum: 0
+          spending_key_hash:
+            type: string
+            format: base16
+            minLength: 56
+            maxLength: 56
+          stake_key_hash:
+            type: string
+            format: base16
+            minLength: 56
+            maxLength: 56
+          script_hash:
+            type: string
+            format: base16
+            minLength: 64
```

Comment:
hmm, I think in 1.20.0 in cardano-node they use 28-byte hash for scripts, not 32-byte

Reply:
Yes, but cardano-addresses still uses 32 bytes :( ... So we need to change it there, bump revision and update here. Which was on my plate today ^^